tools/sense_studio/templates/testing.html

```diff
@@ -0,0 +1,136 @@
+{% extends 'skeleton.html' %}
+
+{% block title %} Testing {% endblock %}
+
+{% block navigation %}
+    {% include 'navigation.html' %}
+{% endblock navigation %}
+
+{% block settings %}
+    {% include 'settings.html' %}
+{% endblock settings %}
+
+{% block main %}
+<div class="uk-container uk-container-xsmall">
+    <h1 class="uk-heading-medium uk-margin-large-top uk-margin-remove-bottom">Testing</h1>
+    <div class="uk-text-meta uk-text-large uk-margin-medium-bottom">
+        <span uk-icon="icon: folder"></span> {{ path }}
+    </div>
+    <form class="uk-form-stacked" method="post">
+        <input type="hidden" id="project" name="project" value="{{ project }}">
+        <input type="hidden" id="path" name="path" value="{{ path }}">
+
+        <div class="uk-child-width-1-1@s uk-grid-small" uk-grid>
+            <div>
+                <label class="uk-form-label" for="classifier">Custom Classifier</label>
+                 <div class="uk-form-controls">
+                    <select class="uk-select {{ 'uk-form-danger' if not classifiers }}" id="classifier">
+                        {% if classifiers %}
+                            {% for classifier in classifiers %}
+                                <option>{{ classifier }}</option>
+                            {% endfor %}
+                        {% else %}
+                            <option value="">No classifiers to select from</option>
+                        {% endif %}
+                    </select>
+                </div>
+            </div>
+        </div>
+
+        <div class="uk-child-width-1-2@s uk-grid-small" uk-grid>
+            <div>
+                <label class="uk-form-label" for="title">Window Title</label>
+                <div class="uk-form-controls">
+                    <div class="uk-inline uk-width-1-1">
+                        <span class="uk-form-icon" uk-icon="icon: pencil"></span>
+                        <input class="uk-input" type="text" id="title" placeholder="Title" autocomplete="off" />
+                    </div>
+                </div>
+            </div>
+            <div class="uk-margin-top">
```

Comment:
Why this margin? Seems to mess up the design
![image](https://user-images.githubusercontent.com/5707898/114717970-e1ad7180-9d35-11eb-9fe6-f0f2ac3ccd54.png)